examples/hotrod/services/driver/redis.go

```diff
@@ -35,27 +35,34 @@ import (
 
 // Redis is a simulator of remote Redis cache
 type Redis struct {
-	tracer opentracing.Tracer // simulate redis as a separate process
+	tracer trace.TracerProvider // simulate redis as a separate process
 	logger log.Factory
 	errorSimulator
 }
 
 func newRedis(otelExporter string, metricsFactory metrics.Factory, logger log.Factory) *Redis {
 	return &Redis{
-		tracer: tracing.Init("redis", otelExporter, metricsFactory, logger),
+		tracer: tracing.InitOTEL("redis", otelExporter, metricsFactory, logger),
 		logger: logger,
 	}
 }
 
 // FindDriverIDs finds IDs of drivers who are near the location.
 func (r *Redis) FindDriverIDs(ctx context.Context, location string) []string {
-	if span := opentracing.SpanFromContext(ctx); span != nil {
+	tracer := r.tracer.Tracer("")
+	if _, span := tracer.Start(ctx, "FindDriverIDs", trace.WithSpanKind(trace.SpanKindClient) /*...set child of option*/); span != nil {
+		span.SetAttributes(attribute.Key("param.location").String(location))
+
+		defer span.End()
+		ctx = trace.ContextWithSpan(ctx, span)
```

Comment:
Doesn't Start already return modified context?

Reply:
Yes, it does, but I thought like L57 `ctx` gets modified with the parent context. but not sure.
https://github.com/jaegertracing/jaeger/blob/bfed6e71ceb633296aab05df10a8fec995c7ff8c/examples/hotrod/services/driver/redis.go#L57

Reply:
This is a difference between OT and OTEL APIs. In OTEL you don't need to do that as it's already done when you call tracer.Start